src/styles/style.scss

```diff
@@ -47,8 +47,6 @@
 @import "../blocks/faq/faq-item/styles/style.scss";
 @import "../blocks/faq/styles/style.scss";
 @import "../blocks/features/styles/style.scss";
-@import "../blocks/food-and-drinks/food-item/styles/style.scss";
-@import "../blocks/food-and-drinks/styles/style.scss";
```

Comment:
These front-end styles need to be restored to maintain the styles of the original block. The editor styles are the only ones not needed anymore.